lib/tape.rb

```diff
@@ -1,5 +1,7 @@
 require 'erb'
 require 'fileutils'
+require 'yaml'
+require_relative 'tape/notifiers/slack.rb'
```

Comment:
Indeed it is but it is odd that `ansible_runner.rb` and `installer.rb` are not also being required here. I've never noticed that....